libkbfs/disk_block_cache_remote.go

```diff
@@ -124,6 +125,20 @@ func (dbcr *DiskBlockCacheRemote) UpdateMetadata(ctx context.Context,
 		})
 }
 
+// GetLastUnrefRev implements the DiskBlockCache interface for
+// DiskBlockCacheRemote.
+func (dbcr *DiskBlockCacheRemote) GetLastUnrefRev(
+	_ context.Context, _ tlf.ID) (kbfsmd.Revision, error) {
+	panic("GetLastUnrefRev() not implemented in DiskBlockCacheRemote")
```

Comment:
I know it's not in use yet in the actual code, but is there a chance that minimal mode could hit this? If so, this should be an error instead of a panic.

Reply:
There is zero chance a remote user will intentionally hit this, so better to panic in my mind.  (Was following the `Status()` function.)

Reply:
Cool thx.